snap/channel/channel.go

```diff
@@ -150,6 +151,16 @@ func (c *Channel) Full() string {
 	return c.String()
 }
 
+// VerbatimTrackOnly returns whether the channel represents a track only.
+func (c *Channel) VerbatimTrackOnly() bool {
```

Comment:
I feel uncomfortable about the `Verbatim` in these. It feels like there's a separate type inside `Channel` trying to get out...

Reply:
I agree, we would need to have different types for ParseVerbatim result vs Parse/Clean, *but* OTOH TBH it feels a bit overkill at this juncture.